packages/stk/stk_io/stk_io/SidesetUpdater.cpp

```diff
@@ -85,9 +85,11 @@ stk::mesh::Entity get_side(const stk::mesh::BulkData& bulk, const stk::mesh::Sid
     return stk::mesh::Entity();
 }
 
-bool is_side_internal(const stk::mesh::BulkData& bulk, stk::mesh::Entity side,
+std::pair<bool,bool> is_side_internal_and_modified(const stk::mesh::BulkData& bulk, stk::mesh::Entity side,
```

Comment:
General advice from the C++ Standard Committee these days is to avoid pairs and tuples in favor of structs with named fields.  e.g.,
```c++
struct InternalOrModified {
  bool isInternal;
  bool isModified;
};

InternalOrModified is_side_internal_or_modified(...) {
  // etc. ....
  return {false, true}; // etc.
}
```

Reply:
Thanks @mhoemmen I like that too. Didn't realize it was general advice these days. I certainly don't like the idiom of using std::tie with returned pairs etc.

Reply:
@alanw0 Structured binding will make this easier -- you won't need to use `std::tie`.